statsmodels/stats/_lilliefors.py

```diff
@@ -216,21 +308,37 @@ def kstest_normal(x, pvalmethod='approx'):
     '''
 
     x = np.asarray(x)
-    z = (x-x.mean())/x.std(ddof=1)
-    nobs = len(z)
+    nobs = len(x)
+
+    if dist == 'norm':
+        z = (x-x.mean())/x.std(ddof=1)
+        test_d = stats.norm.cdf
+        lilliefors_table = get_lilliefors_table(dist='norm')
+    elif dist == 'exp':
+        z = x
+        ex = stats.expon(scale=1./x.mean())
```

Comment:
I meant standardize by scale z = x / x.mean()
Is your scale correct? I'm not sure whether it's x.mean or the reciprocal